# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../euler/nodocinst.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -ruN euler-1.60/source/makefile euler-1.60-new/source/makefile
--- euler-1.60/source/makefile	2002-08-02 16:59:51.000000000 +0200
+++ euler-1.60-new/source/makefile	2004-03-25 18:44:13.815998392 +0100
@@ -163,11 +163,11 @@
 #
 install:
 	if [ ! -d "$(INSTALL_DIR)/share/euler" ]; then mkdir $(INSTALL_DIR)/share/euler ; fi
-	if [ ! -d "$(INSTALL_DIR)/share/doc" ]; then mkdir $(INSTALL_DIR)/share/doc ; fi
-	if [ ! -d "$(INSTALL_DIR)/share/doc/euler" ]; then mkdir $(INSTALL_DIR)/share/doc/euler ; fi
+	#if [ ! -d "$(INSTALL_DIR)/share/doc" ]; then mkdir $(INSTALL_DIR)/share/doc ; fi
+	#if [ ! -d "$(INSTALL_DIR)/share/doc/euler" ]; then mkdir $(INSTALL_DIR)/share/doc/euler ; fi
 	cp -f euler $(INSTALL_DIR)/bin
 	cp -fr ../progs $(INSTALL_DIR)/share/euler
-	cp -fr ../docs/* $(INSTALL_DIR)/share/doc/euler/
+	#cp -fr ../docs/* $(INSTALL_DIR)/share/doc/euler/
 	cp -f help.txt $(INSTALL_DIR)/share/euler
 
 #
